applications/GeoMechanicsApplication/custom_elements/U_Pw_base_element.cpp

```diff
@@ -563,48 +539,12 @@ void UPwBaseElement<TDim, TNumNodes>::CalculateJacobianOnCurrentConfiguration(do
 {
     KRATOS_TRY
 
-    const GeometryType& rGeom = this->GetGeometry();
-    rJ                        = rGeom.Jacobian(rJ, GPoint, mThisIntegrationMethod);
+    rJ = this->GetGeometry().Jacobian(rJ, GPoint, mThisIntegrationMethod);
     MathUtils<double>::InvertMatrix(rJ, rInvJ, detJ);
 
     KRATOS_CATCH("")
 }
 
-//----------------------------------------------------------------------------------------
-template <unsigned int TDim, unsigned int TNumNodes>
-void UPwBaseElement<TDim, TNumNodes>::CalculateJacobianOnCurrentConfiguration(
-    double& detJ, Matrix& J, Matrix& InvJ, Matrix& GradNpT, unsigned int GPoint) const
-{
-    KRATOS_TRY
-
-    const GeometryType& rGeom = this->GetGeometry();
-    Matrix              DisplacementMatrix;
-    GeoElementUtilities::GetNodalVariableMatrix<TDim, TNumNodes>(DisplacementMatrix, rGeom, DISPLACEMENT);
-
-    J.clear();
-    J = this->GetGeometry().Jacobian(J, GPoint, mThisIntegrationMethod, DisplacementMatrix);
-
-    MathUtils<double>::InvertMatrix(J, InvJ, detJ);
-
-    const Matrix& DN_De = this->GetGeometry().ShapeFunctionsLocalGradients(mThisIntegrationMethod)[GPoint];
-    noalias(GradNpT) = prod(DN_De, InvJ);
-
-    KRATOS_CATCH("")
-}
-
-//----------------------------------------------------------------------------------------
-template <unsigned int TDim, unsigned int TNumNodes>
-double UPwBaseElement<TDim, TNumNodes>::CalculateDerivativesOnCurrentConfiguration(
-    Matrix& rJ, Matrix& rInvJ, Matrix& rDN_DX, const IndexType& PointNumber, IntegrationMethod ThisIntegrationMethod) const
-{
-    double detJ;
-    rJ = this->GetGeometry().Jacobian(rJ, PointNumber, ThisIntegrationMethod);
-    const Matrix& DN_De = this->GetGeometry().ShapeFunctionsLocalGradients(ThisIntegrationMethod)[PointNumber];
-    MathUtils<double>::InvertMatrix(rJ, rInvJ, detJ);
-    GeometryUtils::ShapeFunctionsGradients(DN_De, rInvJ, rDN_DX);
-    return detJ;
-}
```

Comment:
I probably just overlooked it since it's quite subtle, but were these versions just not used, while the versions with fewer arguments are?

Reply:
Correct.